test/models/game_test.rb

```diff
@@ -202,7 +202,7 @@ class GameTest < ActiveSupport::TestCase
 
 	test "populate white queen - 1 of 1" do
 		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
-		queen = game.pieces.where(:pos_x => 3).where(:pos_y => 7).first
+		queen = game.pieces.where(:pos_x => 4).where(:pos_y => 7).first
```

Comment:
right, reversed depending on the side...  good call thanks for fixing this test as well.